client/client.go

```diff
@@ -272,56 +281,16 @@ func NewClient(svrAddrs []string, security SecurityOption, opts ...ClientOption)
 	return NewClientWithContext(context.Background(), svrAddrs, security, opts...)
 }
 
-// NewClientWithContext creates a PD client with context.
+// NewClientWithContext creates a PD client with context. This API uses the default keyspace id 0.
 func NewClientWithContext(ctx context.Context, svrAddrs []string, security SecurityOption, opts ...ClientOption) (Client, error) {
-	log.Info("[pd] create pd client with endpoints", zap.Strings("pd-address", svrAddrs))
-	c, clientCtx, clientCancel, tlsCfg := createClient(ctx, 0, &security)
-	// Inject the client options.
-	for _, opt := range opts {
-		opt(c)
-	}
-
-	c.svcDiscovery = newPDServiceDiscovery(clientCtx, clientCancel, &c.wg, addrsToUrls(svrAddrs), tlsCfg, c.option)
-	c.tsoClient = newTSOClient(clientCtx, clientCancel, &c.wg, c.option, c.keyspaceID, c.svcDiscovery, c.svcDiscovery.(tsoAllocatorEventSource), &pdTSOStreamBuilderFactory{})
-	if err := c.setup(); err != nil {
-		c.cancel()
-		return nil, err
-	}
-	if err := c.tsoClient.setup(); err != nil {
-		c.cancel()
-		return nil, err
-	}
-	return c, nil
+	// pass 0 as the default keyspace id
+	return NewClientWithKeyspace(ctx, 0, svrAddrs, security, opts...)
```

Comment:
use constant.go?

Reply:
This client package, and it seems that the client won't use  an server side package -- I have confirmed this with Yutong.

Reply:
but I just defined a constant value in client.go and  use it here.

Changed to NewClientWithKeyspace(ctx, defaultKeyspaceID, svrAddrs, security, opts...)